Items/04_06_MachineGuns_HeavyWeps_&_RocketLaunchers.txt

```diff
@@ -94,6 +94,32 @@ Movement Speed Penalty: -2m
 * 4 out to 45m
 * 5 out to 80m
 
+==Frontier Flame Thrower FLMR-B $3880 ==
+
+Damage: 80 Special
+Mag Size:  8 units of 6 liters, Naphitic Gel. $320 per magazine
+Reload DC:  50
+Fire Modes:  Semi
+Armor Piercing Level: n/a
+Reflex Modifier: -20
+Suppression Level: Medium
+Movement Speed Penalty: -2m
+
+	This flame thrower fires a wide spread of flaming gel, hitting the target 
+space as well as each adjacent space visible to the wielder.

```

Comment:
For clarity, can you catch the square on fire, but miss the target if you don't roll accuracy high enough? What happens to a character that enters, leaves, moves through or ends their turn in a square that is burning? How long does the burning last?

Reply:
I hadn't considered that. I imagine the flame would burn on the ground for one additional turn. I was imagining it as a faster-burning napalm. I'm thinking of making it such that the player rolls dex for each burning space they step in, including the one they were in when the flame thrower hit. Anybody who ends their turn in the fire would roll dex again one more time before the fire diminishes. The burning damage from the fire on the ground would be in the range of 30-40.